pkg/innerring/processors/netmap/process_epoch.go

```diff
@@ -121,6 +125,28 @@ func (np *Processor) updatePlacementInContract(nm netmap.NetMap, l *zap.Logger)
 
 		err = np.containerWrp.UpdateContainerPlacement(cID, vectors, replicas)
 		if err != nil {
+			expBackoff := backoff.NewExponentialBackOff()
+			expBackoff.InitialInterval = 1 * time.Second
+
+			err = backoff.RetryNotify(
+				func() error {
+					err = np.containerWrp.UpdateContainerPlacement(cID, vectors, replicas)
+					if errors.Is(err, client.ErrConnectionLost) {
+						return backoff.Permanent(err)
+					}
+					return err
+				},
+				expBackoff,
+				func(err error, d time.Duration) {
+					l.Warn("retrying updating placement vectors in Container contract",
+						zap.Stringer("cid", cID),
+						zap.Duration("retry-after", d),
```

Comment:
```suggestion
						zap.Stringer("retry-after", d),
```
usually reads more clearly. Although `Duration` is more correct from the type pov